backend/src/SM.Service/Extensions/EventStoreExtensions.cs

```diff
@@ -42,5 +44,28 @@ public static IMessage ReadMessage(this RecordedEvent recordedEvent)
 
             return null;
         }
+
+        public static async Task<PatternItems> GetUserPatterns(this string userId)
```

Comment:
Maybe `GetPatternItems` is a bit better name because that's what the method does.

Reply:
Also no need to create extension methods or every single bit. This extension method will apply for every string type which won't make any sense in all other cases. Better to keep this method in the controller.